ksqldb-parser/src/main/java/io/confluent/ksql/util/ParserUtil.java

```diff
@@ -37,11 +39,15 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.Set;
 import java.util.regex.Pattern;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
 import org.antlr.v4.runtime.ParserRuleContext;
 import org.antlr.v4.runtime.Token;
 import org.antlr.v4.runtime.tree.TerminalNode;
 
+@SuppressWarnings("checkstyle:ClassDataAbstractionCoupling")
```

Comment:
we usually suppress checkstyle warnings with comments instead of `@SuppressWarnings` for example:
```java
  // CHECKSTYLE_RULES.OFF: ClassDataAbstractionCoupling
  public final class ParserUtil {
    // CHECKSTYLE_RULES.ON: ClassDataAbstractionCoupling
```